build/Dockerfile

```diff
@@ -1,18 +1,36 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS builder
-ENV OPERATOR_PATH=/go/src/github.com/openshift/rbac-permissions-operator
-COPY . ${OPERATOR_PATH}
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+
+# Allow specifying a GOPROXY cache during build to speed up dependency resolution
+ARG GOPROXY=https://proxy.golang.org
```

Comment:
This will always ensure that GOPROXY is set later on line 8. Did you mean `ARG GOPROXY`? Drawback to `ARG GOPROXY` is that if the `--build-arg` isn't specified, then `ENV GOPROXY=""` is used.

Reply:
This sets a default value for the argument. If the build-arg is overridden (such as with `export GOPROXY=direct`) then the override will be used.

Reply:
Ahh, I see. I was reading it the other way around where it was setting the proxy to the remote setting is optional. This makes sense.